diff -uraN i3-4.23/i3bar/include/child.h omv-i3-4.23/i3bar/include/child.h
--- i3-4.23/i3bar/include/child.h	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/i3bar/include/child.h	2023-11-01 15:01:10.793124306 +0100
@@ -11,7 +11,7 @@
 
 #include <config.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #define STDIN_CHUNK_SIZE 1024
 
diff -uraN i3-4.23/i3bar/src/child.c omv-i3-4.23/i3bar/src/child.c
--- i3-4.23/i3bar/src/child.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/i3bar/src/child.c	2023-11-01 14:21:05.319168622 +0100
@@ -13,7 +13,7 @@
 #include <ctype.h> /* isspace */
 #include <err.h>
 #include <errno.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <paths.h>
 #include <signal.h>
diff -uraN i3-4.23/i3bar/src/ipc.c omv-i3-4.23/i3bar/src/ipc.c
--- i3-4.23/i3bar/src/ipc.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/i3bar/src/ipc.c	2023-11-01 14:21:18.726168375 +0100
@@ -10,7 +10,7 @@
 #include "common.h"
 
 #include <errno.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <i3/ipc.h>
 #include <stdint.h>
 #include <stdlib.h>
diff -uraN i3-4.23/i3bar/src/main.c omv-i3-4.23/i3bar/src/main.c
--- i3-4.23/i3bar/src/main.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/i3bar/src/main.c	2023-11-01 14:21:30.987168149 +0100
@@ -7,7 +7,7 @@
  */
 #include "common.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <getopt.h>
 #include <glob.h>
 #include <i3/ipc.h>
diff -uraN i3-4.23/i3bar/src/xcb.c omv-i3-4.23/i3bar/src/xcb.c
--- i3-4.23/i3bar/src/xcb.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/i3bar/src/xcb.c	2023-11-01 14:21:47.577167843 +0100
@@ -10,7 +10,7 @@
 #include "common.h"
 
 #include <err.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <i3/ipc.h>
 #include <stdlib.h>
 #include <string.h>
diff -uraN i3-4.23/include/ipc.h omv-i3-4.23/include/ipc.h
--- i3-4.23/include/ipc.h	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/include/ipc.h	2023-11-01 14:22:20.968167228 +0100
@@ -11,7 +11,7 @@
 
 #include <config.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <yajl/yajl_gen.h>
 #include <yajl/yajl_parse.h>
 
diff -uraN i3-4.23/include/log.h omv-i3-4.23/include/log.h
--- i3-4.23/include/log.h	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/include/log.h	2023-11-01 14:23:58.071165439 +0100
@@ -10,7 +10,7 @@
 #pragma once
 
 #include <config.h>
-#include <ev.h>
+#include <libev/ev.h>
 
 /* We will include libi3.h which define its own version of LOG, ELOG.
  * We want *our* version, so we undef the libi3 one. */
diff -uraN i3-4.23/src/ipc.c omv-i3-4.23/src/ipc.c
--- i3-4.23/src/ipc.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/src/ipc.c	2023-11-01 14:22:40.972166860 +0100
@@ -11,7 +11,7 @@
 #include "all.h"
 #include "yajl_utils.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <libgen.h>
 #include <locale.h>
diff -uraN i3-4.23/src/log.c omv-i3-4.23/src/log.c
--- i3-4.23/src/log.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/src/log.c	2023-11-01 14:24:20.250165031 +0100
@@ -12,7 +12,7 @@
 #include "all.h"
 #include "shmlog.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <libgen.h>
 #include <sys/socket.h>
 #include <sys/un.h>
diff -uraN i3-4.23/src/main.c omv-i3-4.23/src/main.c
--- i3-4.23/src/main.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/src/main.c	2023-11-01 14:22:55.466166593 +0100
@@ -10,7 +10,7 @@
 #include "all.h"
 #include "shmlog.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <libgen.h>
diff -uraN i3-4.23/testcases/inject_randr1.5.c omv-i3-4.23/testcases/inject_randr1.5.c
--- i3-4.23/testcases/inject_randr1.5.c	2023-10-29 09:57:51.000000000 +0100
+++ omv-i3-4.23/testcases/inject_randr1.5.c	2023-11-01 14:23:25.560166038 +0100
@@ -14,7 +14,7 @@
  */
 #include "all.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <libgen.h>
