diff -uraN i3-4.24/i3bar/include/child.h omv-i3-4.24/i3bar/include/child.h
--- i3-4.24/i3bar/include/child.h	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/i3bar/include/child.h	2024-12-30 18:20:28.201246549 +0100
@@ -11,7 +11,7 @@
 
 #include <config.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 
 #define STDIN_CHUNK_SIZE 1024
 
diff -uraN i3-4.24/i3bar/src/child.c omv-i3-4.24/i3bar/src/child.c
--- i3-4.24/i3bar/src/child.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/i3bar/src/child.c	2024-12-30 18:20:03.316399011 +0100
@@ -14,7 +14,7 @@
 #include <ctype.h> /* isspace */
 #include <err.h>
 #include <errno.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <paths.h>
 #include <signal.h>
diff -uraN i3-4.24/i3bar/src/ipc.c omv-i3-4.24/i3bar/src/ipc.c
--- i3-4.24/i3bar/src/ipc.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/i3bar/src/ipc.c	2024-12-30 18:32:26.114152942 +0100
@@ -10,7 +10,7 @@
 #include "common.h"
 
 #include <errno.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <i3/ipc.h>
 #include <stdint.h>
 #include <stdlib.h>
diff -uraN i3-4.24/i3bar/src/main.c omv-i3-4.24/i3bar/src/main.c
--- i3-4.24/i3bar/src/main.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/i3bar/src/main.c	2024-12-30 18:20:55.387922663 +0100
@@ -7,7 +7,7 @@
  */
 #include "common.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <getopt.h>
 #include <glob.h>
 #include <i3/ipc.h>
diff -uraN i3-4.24/i3bar/src/xcb.c omv-i3-4.24/i3bar/src/xcb.c
--- i3-4.24/i3bar/src/xcb.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/i3bar/src/xcb.c	2024-12-30 18:21:12.102507627 +0100
@@ -10,7 +10,7 @@
 #include "common.h"
 
 #include <err.h>
-#include <ev.h>
+#include <libev/ev.h>
 #include <i3/ipc.h>
 #include <stddef.h>
 #include <stdlib.h>
diff -uraN i3-4.24/include/ipc.h omv-i3-4.24/include/ipc.h
--- i3-4.24/include/ipc.h	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/include/ipc.h	2024-12-30 18:21:51.969172478 +0100
@@ -11,7 +11,7 @@
 
 #include <config.h>
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <yajl/yajl_gen.h>
 #include <yajl/yajl_parse.h>
 
diff -uraN i3-4.24/include/log.h omv-i3-4.24/include/log.h
--- i3-4.24/include/log.h	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/include/log.h	2024-12-30 18:22:07.534884865 +0100
@@ -10,7 +10,7 @@
 #pragma once
 
 #include <config.h>
-#include <ev.h>
+#include <libev/ev.h>
 
 /* We will include libi3.h which define its own version of LOG, ELOG.
  * We want *our* version, so we undef the libi3 one. */
diff -uraN i3-4.24/src/ipc.c omv-i3-4.24/src/ipc.c
--- i3-4.24/src/ipc.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/src/ipc.c	2024-12-30 18:22:33.496754514 +0100
@@ -11,7 +11,7 @@
 #include "all.h"
 #include "yajl_utils.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <libgen.h>
 #include <locale.h>
diff -uraN i3-4.24/src/log.c omv-i3-4.24/src/log.c
--- i3-4.24/src/log.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/src/log.c	2024-12-30 18:22:47.013653593 +0100
@@ -12,7 +12,7 @@
 #include "all.h"
 #include "shmlog.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <libgen.h>
 #include <sys/socket.h>
 #include <sys/un.h>
diff -uraN i3-4.24/src/main.c omv-i3-4.24/src/main.c
--- i3-4.24/src/main.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/src/main.c	2024-12-30 18:23:10.895721707 +0100
@@ -10,7 +10,7 @@
 #include "all.h"
 #include "shmlog.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <libgen.h>
diff -uraN i3-4.24/testcases/inject_randr1.5.c omv-i3-4.24/testcases/inject_randr1.5.c
--- i3-4.24/testcases/inject_randr1.5.c	2024-11-06 18:26:52.000000000 +0100
+++ omv-i3-4.24/testcases/inject_randr1.5.c	2024-12-30 18:23:33.716890820 +0100
@@ -14,7 +14,7 @@
  */
 #include "all.h"
 
-#include <ev.h>
+#include <libev/ev.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <libgen.h>
