--- i3-4.3/i3-config-wizard/i3-config-wizard.mk.libev	2012-09-19 20:08:09.000000000 +0400
+++ i3-4.3/i3-config-wizard/i3-config-wizard.mk	2012-11-06 21:00:14.347821406 +0400
@@ -5,7 +5,7 @@ CLEAN_TARGETS += clean-i3-config-wizard
 i3_config_wizard_SOURCES_GENERATED  = i3-config-wizard/cfgparse.tab.c i3-config-wizard/cfgparse.yy.c
 i3_config_wizard_SOURCES           := $(filter-out $(i3_config_wizard_SOURCES_GENERATED),$(wildcard i3-config-wizard/*.c))
 i3_config_wizard_HEADERS           := $(wildcard i3-config-wizard/*.h)
-i3_config_wizard_CFLAGS             = $(XCB_CFLAGS) $(XCB_KBD_CFLAGS) $(X11_CFLAGS) $(PANGO_CFLAGS)
+i3_config_wizard_CFLAGS             = $(XCB_CFLAGS) $(XCB_KBD_CFLAGS) $(X11_CFLAGS) $(PANGO_CFLAGS) $(LIBEV_CFLAGS)
 i3_config_wizard_LIBS               = $(XCB_LIBS) $(XCB_KBD_LIBS) $(X11_LIBS) $(PANGO_LIBS)
 
 i3_config_wizard_OBJECTS := $(i3_config_wizard_SOURCES_GENERATED:.c=.o) $(i3_config_wizard_SOURCES:.c=.o)
--- i3-4.3/i3-input/i3-input.mk.libev	2012-09-19 20:08:14.000000000 +0400
+++ i3-4.3/i3-input/i3-input.mk	2012-11-06 21:00:14.347821406 +0400
@@ -4,7 +4,7 @@ CLEAN_TARGETS += clean-i3-input
 
 i3_input_SOURCES := $(wildcard i3-input/*.c)
 i3_input_HEADERS := $(wildcard i3-input/*.h)
-i3_input_CFLAGS   = $(XCB_CFLAGS) $(XCB_KBD_CFLAGS) $(PANGO_CFLAGS)
+i3_input_CFLAGS   = $(XCB_CFLAGS) $(XCB_KBD_CFLAGS) $(PANGO_CFLAGS) $(LIBEV_CFLAGS)
 i3_input_LIBS     = $(XCB_LIBS) $(XCB_KBD_LIBS) $(PANGO_LIBS)
 
 i3_input_OBJECTS := $(i3_input_SOURCES:.c=.o)
--- i3-4.3/i3-msg/i3-msg.mk.libev	2012-09-19 20:08:09.000000000 +0400
+++ i3-4.3/i3-msg/i3-msg.mk	2012-11-06 21:00:14.347821406 +0400
@@ -4,7 +4,7 @@ CLEAN_TARGETS += clean-i3-msg
 
 i3_msg_SOURCES := $(wildcard i3-msg/*.c)
 i3_msg_HEADERS := $(wildcard i3-msg/*.h)
-i3_msg_CFLAGS   = $(XCB_CFLAGS) $(PANGO_CFLAGS)
+i3_msg_CFLAGS   = $(XCB_CFLAGS) $(PANGO_CFLAGS) $(LIBEV_CFLAGS)
 i3_msg_LIBS     = $(XCB_LIBS)
 
 i3_msg_OBJECTS := $(i3_msg_SOURCES:.c=.o)
--- i3-4.3/i3-nagbar/i3-nagbar.mk.libev	2012-09-19 20:08:09.000000000 +0400
+++ i3-4.3/i3-nagbar/i3-nagbar.mk	2012-11-06 21:00:59.945780909 +0400
@@ -4,7 +4,7 @@ CLEAN_TARGETS += clean-i3-nagbar
 
 i3_nagbar_SOURCES := $(wildcard i3-nagbar/*.c)
 i3_nagbar_HEADERS := $(wildcard i3-nagbar/*.h)
-i3_nagbar_CFLAGS   = $(XCB_CFLAGS) $(PANGO_CFLAGS)
+i3_nagbar_CFLAGS   = $(XCB_CFLAGS) $(PANGO_CFLAGS) $(LIBEV_CFLAGS)
 i3_nagbar_LIBS     = $(XCB_LIBS) $(PANGO_LIBS)
 
 i3_nagbar_OBJECTS := $(i3_nagbar_SOURCES:.c=.o)
